source/includes/_accounts.md

```diff
@@ -15,6 +15,102 @@ curl -X POST -H "Content-Type: application/json" -H "Cache-Control: no-cache" -d
 }' "https://api.apollo.io/v1/accounts/search"
 ```
 
+> Sample response:
+
+```json
+{
+    "breadcrumbs": [
+        {
+            "label": "Company Name",
+            "signal_field_name": "q_organization_name",
+            "value": "Google",
+            "display_name": "Google"
+        }
+    ],
+    "partial_results_only": false,
+    "disable_eu_prospecting": false,
+    "partial_results_limit": 10000,
+    "pagination": {
+        "page": 1,
+        "per_page": 25,
+        "total_entries": 2,
+        "total_pages": 1
+    },
+    "accounts": [
+        {
+            "id": "5c3d15XXXXXXXXXXXXXXXXXX",
+            "name": "Google",
+            "website_url": "http://www.google.com",
+            "blog_url": null,
+            "angellist_url": "http://angel.co/google",
+            "linkedin_url": "http://www.linkedin.com/company/google",
+            "twitter_url": "http://twitter.com/google",
+            "facebook_url": "https://www.facebook.com/Google",
+            "primary_phone": {
```

Comment:
get rid of this field in api doc (there is already a field called "phone". We should not disclose our `source` here.)